cmake_files/dependencies.cmake

```diff
@@ -24,7 +20,6 @@ endif()
 
 
 if(regressions)
-  find_package(FFTW3 REQUIRED DOUBLE)
```

Comment:
is `fftw3` only used for the regressions? I wouldn't set it as a global dependency.

Reply:
@SJaffa and I liked the `conanfile.py` idea and added one in [e926cab](https://github.com/astro-informatics/sopt/pull/265/commits/e926cabc0708bf49ea6668f36ee09a4da7919c3d)